TUM-Dev/NavigaTUM
# initial migration of search to utilise init containers

Fixes #352 

## Proposed Changes (include Screenshots if possible)

- Changed search to use initcontainers instead of rebuilding in gitlab actions

## How to test this PR

1. Launch the website
2. See if search still works

## How has this been tested?

- TODO

## Checklist:
